README.md

```diff
@@ -8,30 +8,5 @@ The image is signed and attested by Tekton Chains. For verification, use the
 
 # To build in AppStudio
 
-Best use `hack/rebuild.sh` from [ec-cli](https://github.com/hacbs-contract/ec-cli). To build manually create the following `PipelineRun`:
-
-```yaml
```

Comment:
Removing this because the current version is not even accurate. We don't really need to duplicate the yaml here. Added text to point to the actual yaml that is used.